spring-cloud-data-rest-resource/src/main/java/org/springframework/cloud/data/rest/resource/ModuleRegistrationResource.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
all of the copyright headers should have 5 spaces before "http:" (recently a bulk update took care of most, we all need to ensure our templates are up-to-date)